src/screens/settings/container/settingsContainer.tsx

```diff
@@ -305,7 +305,19 @@ class SettingsContainer extends Component {
         break;
 
       case settingsTypes.BACKUP_PRIVATE_KEYS:
-        navigation.navigate(ROUTES.SCREENS.BACKUP_KEYS);
+        if (isPinCodeOpen) {
```

Comment:
@aliseyalvi please redo this part, the current implementation will,

- make users not using pinCode unable to access manage screen
- break functionality for those already using pinCode as your configuration for pinCode screen will put it in reset mode...

for reference check code here: https://github.com/ecency/ecency-mobile/blob/a93b36b1d1705e4b58211a33319d78978461a2ea/src/screens/assetDetails/screen/assetDetailsScreen.tsx#L147